src/mongoose_api_client.erl

```diff
@@ -63,6 +63,8 @@ cowboy_router_paths(Base, _Opts) ->
 
 init(Req, Opts) ->
     Bindings = maps:to_list(cowboy_req:bindings(Req)),
+    ?LOG_ERROR(#{what => rest_init,
+                 cowboy_req => Req}),
```

Comment:
`ERROR`? This seems like a debug left-over

Reply:
leftovers